src/storer.ts

```diff
@@ -17,16 +17,8 @@ export class Storer {
 
     constructor(){
         //Connect to database using api's index
-        require('../api/database').connect(null, context => {
-            //Store context
-            this.context = context;
-            /*
-             Make sure that syncing to database is synchronous.
-             Not that there is no {force: true} option here: We don't want to overwrite
-             existing tables.
-             */
-            context.sequelize.sync().then(() => {return this;});
-        });
+        this.context = require('../api/database').getContext();
```

Comment:
we connect to the DB in the processing manager BUT we don't do this in workers. Workers are completely different processes